doc/website/blog/tbd-announcing-versioned-migrations.md

```diff
@@ -0,0 +1,369 @@
+---
+title: Announcing Versioned Migrations
+author: MasseElch
+authorURL: "https://github.com/masseelch"
+authorImageURL: "https://avatars.githubusercontent.com/u/12862103?v=4"
+image: tbd
+---
+
+When [Ariel](https://github.com/a8m) released Ent v0.10.0 at the end of January,
+he [introduced](2022-01-20-announcing-new-migration-engine.md) a new migration engine for Ent based on another
+open-source project called [Atlas](https://github.com/ariga/atlas). 
+
+Initially, Atlas supported a style of managing database schemas that we call "declarative migrations". With declarative
+migrations, the desired state of the database schema is given as input to the migration engine, which plans and executes
+a set of actions to change the database to its desired state. This approach has been popularized in the field of
+cloud native infrastructure by projects such as Kubernetes and Terraform. It works great in many cases, in
+fact it has served the Ent framework very well in the past few years. However, database migrations are a very sensitive
+topic, and many projects require a more controlled approach.
+
+For this reason, most industry standard solutions, like [Flyway](https://flywaydb.org/)
+, [Liquibase](https://liquibase.org/), or [golang-migrate/migrate](https://github.com/golang-migrate/migrate) (which is
+common in the Go ecosystem), support a workflow that they call "versioned migrations".
+
+With versioned migrations, you assign each migration (commonly a set of SQL files) a unique version and a description
+and check those files into version control together with your application code. Versioned migrations are then applied in
+order exactly once on your database.
+
+> Liquibase is an open-source database schema change management solution which enables you to manage revisions of your 
+> database changes easily. Liquibase makes it easy for anyone involved in the application release process to:
+>
+> - Eliminate errors and delays when releasing databases.
+> - Deploy and roll back changes for specific versions without needing to know what has already been deployed.
+> - Deploy database and application changes together so they always stay in sync.
```

Comment:
Needs some segue into the self-description of another project. Also, I would take a definition of versioned migrations or some description of the mechanism instead of the general "marketing copy" of the project. The point is to use others' words to explain the concept of versioned migrations.

Reply:
They don't really have a description :cry:

Reply:
@rotemtam I added a description of it in my own words now. I cannot find a good explanation on the mentioned tools' docs.